src/mcts/search.h

```diff
@@ -103,10 +104,16 @@ class Search {
   static const char* kPolicySoftmaxTempStr;
   static const char* kAllowedNodeCollisionsStr;
   static const char* kStickyCheckmateStr;
+  static const char* kMultiPvStr;
 
  private:
   // Returns the best move, maybe with temperature (according to the settings).
   std::pair<Move, Move> GetBestMoveInternal() const;
+  
+  // Returns a list of all children with, with or without temperature in descending order of visits.
+  std::list<EdgeAndNode> GetAscChildrenNoTemperature(Node* parent) const;
+  std::list<EdgeAndNode> GetAscChildrenWithTemperature(Node* parent,
```

Comment:
Kind of unnecessary (not to mention meaningless) function, per Discord discussion